leadership/README.md

```diff
@@ -57,3 +55,30 @@ for leader := <-leaderCh {
 
 A typical use case for this is to be able to always send requests to the current
 leader.
+
+## Fault tolerance
+
+Leadership returns an error channel for Candidates and Followers that you can use
```

Comment:
@aluzzardi I think errCh helps calling `underwood.RunForElection()` only once and outside the loop,

otherwise we'd be asking for an `electedCh` every single time inside the `for{}`. @abronan, is that right ?